daemon/cmd/daemon.go

```diff
@@ -275,8 +278,6 @@ func (d *Daemon) init() error {
 	}
 
 	if !option.Config.DryMode {
-		bandwidth.InitBandwidthManager()
```

Comment:
It seems like `DryMode` is not going to work with new implementation as `init()` will always be called and will create maps etc (or maybe I just missed something)?

Reply:
The map is only created when it is used. You are right that the `init` would run regardless of the dry mode. I changes the cell so we only register the start hook when not in dry mode. That effectively does the same as the old version.